data/language/en-GB.txt

```diff
@@ -4199,6 +4199,8 @@ STR_5887    :Prefix
 STR_5888    :Suffix
 STR_5889    :Custom currency symbol
 STR_5890    :Enter the currency symbol to display
+STR_5898    :Default
```

Comment:
I presume it has to do with #3964, but that PR has since added a few more strings.